plugin/path_login.go

```diff
@@ -399,24 +405,27 @@ func (b *GcpAuthBackend) pathGceLogin(req *logical.Request, loginInfo *gcpLoginI
 		return logical.ErrorResponse(err.Error()), nil
 	}
 
+	iamClient, err := b.IAM(req.Storage)
```

Comment:
Can we remove `fmt.Printf()` from lines 379 and 380?

Reply:
Yup!